library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -1145,6 +1148,7 @@ public void writeToParcel(@NonNull Parcel out, int flags) {
             out.writeInt(dynamicHeightEnabled ? 1 : 0);
             out.writeInt(calendarMode == CalendarMode.WEEKS ? 1 : 0);
             out.writeParcelable(currentMonth, 0);
+            out.writeByte((byte) (saveCurrentPosition ? 1 : 0));
```

Comment:
It appears to only have boolean arrays, not explicit boolean. I did this the same way it was done for allowClickDaysOutsideCurrentMonth on line 1139